ambry-clustermap/src/main/java/com/github/ambry/clustermap/HelixClusterManager.java

```diff
@@ -451,8 +472,12 @@ public ReplicaId getBootstrapReplica(String partitionIdStr, DataNodeId dataNodeI
 
   @Override
   public void registerClusterMapListener(ClusterMapChangeListener clusterMapChangeListener) {
-    for (DcInfo dcInfo : dcToDcInfo.values()) {
-      dcInfo.clusterChangeHandler.registerClusterMapListener(clusterMapChangeListener);
+    if (clusterMapConfig.clusterMapUseAggregatedView) {
+      helixAggregatedViewClusterInfo.clusterChangeHandler.registerClusterMapListener(clusterMapChangeListener);
+    } else {
+      for (DcInfo helixDcInfo : dcToDcInfo.values()) {
+        helixDcInfo.clusterChangeHandler.registerClusterMapListener(clusterMapChangeListener);
```

Comment:
What does registerClusterMapListener listen to?  In the above #1 to #4, if it listens to #3 and #4, does it needs to listen to changes in each DC?